pkg/ccl/backupccl/backup_test.go

```diff
@@ -1464,6 +1464,7 @@ func checkInProgressBackupRestore(
 	defer cleanup()
 
 	sqlDB.Exec(t, `CREATE DATABASE restoredb`)
+	sqlDB.Exec(t, `SET CLUSTER SETTING bulkio.backup.merge_file_size = '0'`)
```

Comment:
Worthwhile a comment here so that we remember why this was needed in future?

Reply:
Yep, good point. Done.